doc/releases/changelog-dev.md

```diff
@@ -56,6 +56,10 @@
   Instead, use `pennylane.gradients.classical_fisher` and `pennylane.gradients.quantum_fisher`.
   [(#5985)](https://github.com/PennyLaneAI/pennylane/pull/5985)
 
+* The legacy devices `default.qubit.{default.qubit.{legacy,autograd,torch,tf,jax}}` are deprecated.
```

Comment:
```suggestion
* The legacy devices `default.qubit.{default.qubit.{autograd,torch,tf,jax,legacy}}` are deprecated.
```
Just to mention `legacy` last because I believe it is the least directly used.